specs/vep-1739-multiple-python-versions/diagrams/high_level_sequence_plantUML_source.txt

```diff
@@ -0,0 +1,21 @@
+@startuml
+actor User as engineer
+participant ControlService as cs
+participant K8sNamespace_DataJobBuilders as namespace
+database Database as db
+database Registry as ecr
+
+engineer -> cs : vdk create
+cs -> db : Register new data job
+engineer -> engineer : Set <b>python_version</b> in the data job's config.ini
+engineer -> cs : vdk deploy
+cs -> cs : Read Deployment data
+cs -> db : Update data job configuration data
+
+cs -> ecr : Request data job base image
+activate ecr
+cs -> ecr : Request vdk image
+return Return images
+
+cs -> namespace : Start data job builder
```

Comment:
I suggest we remove the create workflow as well, it's not something new.
Let's make where we set the python version explicitly clear. 

Also, it's a good idea to indicate what happens synchronously to the user currently and what happens async. 

basically we store the new requests python version synchronously but then the actual deployment process is async. 

```suggestion
engineer -> engineer : Set <b>python_version</b> in the data job's config.ini
engineer -> cs : vdk deploy
activate cs
cs -> cs : Read Deployment data
cs -> db : Update data job configuration data with specified python version
return deployment started 
deactivate cs

cs ->> cs : Start deployment process (async)
cs -> ecr : Request data job base image with specified python version
activate ecr
cs -> ecr : Request vdk image with specified python version
return Return images

cs -> namespace : Start data job builder
```

Reply:
Updated the graph, thanks